README.md

```diff
@@ -345,6 +347,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
     * [**Forkgram**](https://github.com/Forkgram/TelegramAndroid) <sup>**[[F-Droid](https://f-droid.org/app/org.forkgram.messenger)]**</sup>
 * [**Element**](https://github.com/vector-im/element-android) <sup>**[[F-Droid](https://f-droid.org/app/im.vector.app)]**</sup>
 * [**Signal**](https://github.com/signalapp/Signal-Android)
+    * [**Molly**](https://github.com/mollyim/mollyim-android)
```

Comment:
It seems to provide a Free build flavours, which exclude GMS, Play, Firebase etc. :+1: